lib/senders/index.js

```diff
@@ -160,7 +151,7 @@ module.exports = function (log, config, error, bounces, translator, sender) {
       },
       sendPasswordChangedNotification: function (emails, account, opts) {
         var primaryEmail = account.email
-        var ccEmails = getSecondaryEmails(emails)
+        var ccEmails = getVerifiedSecondaryEmails(emails)
```

Comment:
Also tangential: Seems like there's a lot of common boilerplate in these methods. Worth opening an issue to fix it?